src/cryptonote_core/blockchain.cpp

```diff
@@ -627,9 +627,10 @@ void Blockchain::pop_blocks(uint64_t nblocks)
     const uint64_t blockchain_height = m_db->height();
     if (blockchain_height > 0)
       nblocks = std::min(nblocks, blockchain_height - 1);
-    for (i=0; i < nblocks; ++i)
+    while (i < nblocks)
```

Comment:
Didn't like `for ( ; i < nblocks; ++i)` ?

Reply:
I do not think I tried that.